views/emails/html/new-message.php

```diff
@@ -121,37 +118,34 @@
                 <td style="padding: 0 50px 50px;">
                     <div>
                         <h2 style="font-size: 20px; color: #000000; letter-spacing: 0.18px; line-height: 30px; margin-bottom: 15px;">
-                            <?php _e( 'Discussion Members', 'wedevs-project-manager'); ?>
+                            <?php esc_html_e( 'Discussion Members', 'wedevs-project-manager'); ?>
                         </h2>
                         <span style="margin: 0; padding: 0; list-style: none; display: inline-block;">
 
                              <?php
-                                
                                 foreach( $project['assignees']['data'] as $assign ) {
                                     ?>
 
-                                        <img 
+                                        <img
                                             style="
-                                                outline:none; 
-                                                margin-right: 4px; 
-                                                text-decoration:none; 
-                                                height: 33px; 
-                                                width: 33px; 
-                                                border-radius: 50%;" 
+                                                outline:none;
+                                                margin-right: 4px;
+                                                text-decoration:none;
+                                                height: 33px;
+                                                width: 33px;
+                                                border-radius: 50%;"
 
-                                            src="<?php echo esc_url( $assign->avatar_url ); ?>" 
-                                            alt="<?php echo $assign->display_name; ?>" title="<?php echo $assign->display_name; ?>" width="33" height="33" 
+                                            src="<?php echo esc_url( $assign->avatar_url ); ?>"
+                                            alt="<?php echo esc_html( $assign->display_name ); ?>" title="<?php echo esc_html( $assign->display_name ); ?>" width="33" height="33"
```

Comment:
Use `esc_attr` as these values are being used as the values of attributes.
`alt="<?php echo esc_attr( $assign->display_name ); ?>" title="<?php echo esc_attr( $assign->display_name ); ?>" width="33" height="33"`